common/workunit/workunit.cpp

```diff
@@ -8626,6 +8776,11 @@ IConstWUScopeIterator & CLocalWorkUnit::getScopeIterator(const WuScopeFilter & f
             compoundIter->addIter(workflowIter);
         }
     }
+    //if (sources & SSFsearchWorkflow)
```

Comment:
Should check sources & SSFsearchExceptions.  But will need to check that flag is correctly set up from the filter.  E.g. in finishedFilter, inside the if (!sourceFlags):

if (!(properties & (PTnotes)))
         sourceFlags &= ~(SSFsearchExceptions);